JSONSchema/genesis-filler-schema.json

```diff
@@ -0,0 +1,134 @@
+{
+    "definitions": {
+        "MiningMethod": {
+            "anyOf": [
+                {
+                    "pattern": "NoProof",
+                    "type": "string"
+                }
+            ]
+        },
+        "PreStateAccount": {
+            "additionalProperties": true,
+            "properties": {
+                "balance": {
+                    "$ref": "#/definitions/IntegerOrConfusedHex"
+                },
+                "code": {
+                    "$ref": "#/definitions/HexDataOrEmpty"
+                },
+                "nonce": {
+                    "$ref": "#/definitions/IntegerOrConfusedHex"
+                },
+                "storage": {
+                    "additionalProperties": false,
+                    "patternProperties": {
+                        "^(0x)?[0-9a-f]+": {
+                            "$ref": "#/definitions/HexData",
+                            "description": "storage key with 0x. data is HexData"
+                        }
+                    },
+                    "type": "object"
+                }
+            },
+            "required": [
+                "balance",
+                "code",
+                "nonce",
+                "storage"
+            ],
+            "type": "object"
+        }
+    },
+    "patternProperties": {
+        "^.*$": {
+            "properties": {
+                "_info": {
+                    "$ref": "#/definitions/FillerInfo"
+                },
+                "accounts": {
+                    "additionalProperties": false,
+                    "patternProperties": {
+                        "^(0x)?[0-9a-f]*": {
+                            "$ref": "#/definitions/PreStateAccount",
+                            "description": "prestate account address with 0x prefix"
+                        }
+                    },
+                    "type": "object"
+                },
+                "genesis": {
+                    "additionalProperties": false,
+                    "properties": {
+                        "author": {
+                            "$ref": "#/definitions/AddressMaybePrefixOrEmpty"
+                        },
+                        "difficulty": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "extraData": {
+                            "$ref": "#/definitions/HexData"
+                        },
+                        "gasLimit": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "nonce": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "timestamp": {
```

Comment:
Might make sense to add an optional `number` or `blockNumber` to the genesis. Particularly if we want to go with EIP 210, blockhash, it will be good to run tests on high numbered blocks.